diff --git a/dist/zod-DSgpEGAE.js b/dist/zod-DSgpEGAE.js
index 5bc5a98c7763ebf11f223fb462629533ceee644c..9984c62f1c6ec4784837f08972199317d2baf7f7 100644
--- a/dist/zod-DSgpEGAE.js
+++ b/dist/zod-DSgpEGAE.js
@@ -5,12 +5,6 @@ import './vendors/convert.js';
 
 function getToOpenAPISchemaFn() {
   return async (schema, context) => {
-    if ("_zod" in schema) {
-      return getToOpenAPISchemaFn$1()(schema, {
-        components: context.components,
-        options: { io: "input", ...context.options }
-      });
-    }
     try {
       const { createSchema } = await import('zod-openapi');
       const { schema: _schema, components } = createSchema(
diff --git a/dist/zod-DrFzROei.cjs b/dist/zod-DrFzROei.cjs
index a0797b5176f47dcb9aa13460fb668130176a22ca..8763b218de6fe3e822f1088a57874fb97d2959e4 100644
--- a/dist/zod-DrFzROei.cjs
+++ b/dist/zod-DrFzROei.cjs
@@ -7,12 +7,6 @@ require('./vendors/convert.cjs');
 
 function getToOpenAPISchemaFn() {
   return async (schema, context) => {
-    if ("_zod" in schema) {
-      return _default.default()(schema, {
-        components: context.components,
-        options: { io: "input", ...context.options }
-      });
-    }
     try {
       const { createSchema } = await import('zod-openapi');
       const { schema: _schema, components } = createSchema(
